client/my-sites/media-library/content.jsx

```diff
@@ -276,6 +276,10 @@ export class MediaLibraryContent extends Component {
 			);
 		}
 
+		if ( source === 'openverse' ) {
+			return translate( 'We were unable to connect to Openverse. Please try again later.' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'We were unable to connect to the external service. Please try again later.' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/external-media/openverse)